data/api_country_overview_yearly.csv

```diff
@@ -3024,28 +3024,29 @@ Montserrat,MSR,2018,0.02,4.59,500.0,North America,Latin America and Caribbean,0.
-Morocco,MAR,2013,32.18,0.95,654.59,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2021,2040,2040
-Morocco,MAR,2014,33.75,0.99,700.43,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2021,2040,2040
-Morocco,MAR,2015,34.5,1.0,688.79,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2021,2040,2040
-Morocco,MAR,2016,35.63,1.02,681.76,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2021,2040,2040
-Morocco,MAR,2017,37.47,1.06,685.14,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2021,2040,2040
-Morocco,MAR,2018,37.6,1.05,666.96,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2021,2040,2040
-Morocco,MAR,2019,39.06,1.08,678.17,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2021,2040,2040
-Morocco,MAR,2020,38.61,1.05,693.85,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2021,2040,2040
-Morocco,MAR,2021,42.27,1.14,680.63,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2021,2040,2040
+Morocco,MAR,2000,14.13,0.5,856.55,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2001,15.65,0.54,845.07,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2002,16.5,0.56,827.72,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2003,17.9,0.6,796.12,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2004,19.0,0.63,793.6,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2005,19.58,0.64,783.81,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2006,21.26,0.69,780.37,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2007,22.8,0.73,772.61,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2008,23.9,0.76,758.66,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2009,24.65,0.77,689.47,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2010,27.58,0.85,664.55,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2011,29.61,0.9,703.2,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2012,32.24,0.97,698.54,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2013,32.18,0.95,654.59,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2014,33.75,0.99,700.43,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2015,34.5,1.0,688.79,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2016,35.63,1.02,681.76,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2017,37.47,1.06,685.14,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2018,37.6,1.05,666.96,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2019,39.06,1.08,678.17,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2020,38.61,1.05,693.85,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2021,42.27,1.14,680.63,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
+Morocco,MAR,2022,43.84,1.18,705.67,Africa,Africa,0.0,0.0,0.0,0.0,60.0,4.0,,,2022,2040,2040
```

Comment:
No changes to MAR except for 2022. There is no record for Africa in 2022 - hence the lack of update for Africa.